src/network/html/app.css

```
@@ -0,0 +1,529 @@
+h3 {
+  color: var(--foreground);
+  font-size: 22px;
+  line-height: 1.15;
+  font-weight: 700;
+}
+
+svg {
+  width: 16px;
+  height: 16px;
+  display: block;
+  stroke: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)